<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="SonaliDas" ignored="0" name="TC_06" tests="1" failures="1" timestamp="10 Jul 2018 07:42:00 GMT" time="0.661" errors="0">
  <testcase name="@AfterMethod afterMethod" time="0.023" classname="com.tata.selenium.test.deliveryStatistics.TC_04_SelectCustomerAccount">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at com.tata.selenium.test.deliveryStatistics.TC_04_SelectCustomerAccount.afterMethod(TC_04_SelectCustomerAccount.java:290)
... Removed 28 stack frames]]>
    </failure>
  </testcase> <!-- @AfterMethod afterMethod -->
  <testcase name="DO" time="0.545" classname="com.tata.selenium.test.deliveryStatistics.TC_04_SelectCustomerAccount">
    <failure type="java.lang.AssertionError" message="Error occured while launching the url -   java.net.MalformedURLException: For input string: &amp;quot;null&amp;quot;">
      <![CDATA[java.lang.AssertionError: Error occured while launching the url -   java.net.MalformedURLException: For input string: "null"
at com.tata.selenium.utils.CommonUtils.LaunchUrl(CommonUtils.java:214)
at com.tata.selenium.test.deliveryStatistics.TC_04_SelectCustomerAccount.DO(TC_04_SelectCustomerAccount.java:89)
... Removed 26 stack frames]]>
    </failure>
  </testcase> <!-- DO -->
</testsuite> <!-- TC_06 -->
